frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -107,44 +88,55 @@ export default {
     }
   },
   methods: {
-    initScrollbar() {
-      const isWindows = navigator.platform.startsWith('Win')
-      if (isWindows) {
-        initScrollbar('sidenav')
-      }
-    },
     async logout() {
-      await loginAPI.logout(this.$store.state.sessionId)
-      // do we have to check success?
-      this.$sidebar.displaySidebar(false)
-      this.$store.dispatch('logout')
-      this.$router.push('/login')
+      this.$apollo
+        .query({
+          query: logout,
+          variables: { sessionId: this.$store.state.sessionId },
+        })
+        .then(() => {
+          this.$sidebar.displaySidebar(false)
+          this.$store.dispatch('logout')
+          this.$router.push('/login')
+        })
+        .catch(() => {
+          this.$sidebar.displaySidebar(false)
+          this.$store.dispatch('logout')
+          this.$router.push('/login')
+        })
     },
     async updateTransactions(pagination) {
       this.pending = true
-      const result = await communityAPI.transactions(
-        this.$store.state.sessionId,
-        pagination.firstPage,
-        pagination.items,
-      )
-      if (result.success) {
-        this.GdtBalance = Number(result.result.data.gdtSum)
-        this.transactions = result.result.data.transactions
-        this.balance = Number(result.result.data.decay)
-        this.bookedBalance = Number(result.result.data.balance)
-        this.transactionCount = result.result.data.count
-        this.pending = false
-      } else {
-        this.pending = true
-        this.$toasted.error(result.result.message)
-      }
+      this.$apollo
+        .query({
+          query: transactionsQuery,
+          variables: {
+            sessionId: this.$store.state.sessionId,
+            firstPage: pagination.firstPage,
+            items: pagination.items,
+          },
+          fetchPolicy: 'network-only',
+        })
+        .then((result) => {
+          const {
+            data: { transactionList },
+          } = result
+          this.GdtBalance = Number(transactionList.gdtSum)
+          this.transactions = transactionList.transactions
+          this.balance = Number(transactionList.decay)
+          this.bookedBalance = Number(transactionList.balance)
+          this.transactionCount = transactionList.count
+          this.pending = false
+        })
+        .catch((error) => {
+          this.pending = true
```

Comment:
this looks wrong. If so, can we have a test for this?

Reply:
We have to clarify the behavior here. When loading of the transaction list fails, we have an issue